stack-snapshot.yaml

```diff
@@ -12,6 +12,7 @@ packages:
     sha256: "cf39f2b378485afc77ffdad4dbb057d5d9b4dfc5a38c76ddc44e920e537fb0fa"
   - ghc-lib-parser-ex-8.8.5.8
   - bytestring-nums-0.3.6
+  - directory-1.3.6.1
```

Comment:
you have to repin on both linux and windows for this to have any effect

Reply:
thanks!

Reply:
thanks!